build/patches/Enable-darken-websites-checkbox-in-themes.patch

```diff
@@ -11,26 +11,26 @@ Unexpire #darken-websites-checkbox-in-themes-setting
 diff --git a/chrome/browser/flag-metadata.json b/chrome/browser/flag-metadata.json
 --- a/chrome/browser/flag-metadata.json
 +++ b/chrome/browser/flag-metadata.json
-@@ -1031,7 +1031,7 @@
+@@ -949,7 +949,7 @@
    {
-     "name": "darken-websites-checkbox-in-themes-setting",
-     "owners": [ "lazzzis@google.com", "twellington" ],
+     "name": "continuous-search",
```

Comment:
you are editing the wrong flag. it should be `darken-websites-checkbox-in-themes-setting`

Reply:
Also this one, caused by fuzz patch application. Will exclude this file in future.